app/templates/plans.hbs

```diff
@@ -1,69 +1,132 @@
-          <li>{{svg-jar 'icon-passed' class="w-4"}} Unlimited repositories</li>
-          <li>{{svg-jar 'icon-passed' class="w-4"}} Unlimited collaborators</li>
-        </ul>
-      </li>
-      <li class="plan--important">
-        <p class="plans-price">$129</p>
-        <p class="plans-label plans-timeframe">per month</p>
-        <h2 class="plans-name">Startup</h2>
-        <h3 class="plans-label">Best for small teams</h3>
-        <ul class="list">
-          <li><span class="list-em">2</span>Concurrent jobs</li>
-          <li>{{svg-jar 'icon-passed' class="w-4"}} Unlimited build minutes</li>
-          <li>{{svg-jar 'icon-passed' class="w-4"}} Unlimited repositories</li>
-          <li>{{svg-jar 'icon-passed' class="w-4"}} Unlimited collaborators</li>
-        </ul>
-        <div class="align-center">
-          <a href="https://travis-ci.com/signin" class="plans-button--blue" {{action 'gaCta' 'plans-table'}}>Start Trial</a>
-        </div>
-      </li>
+  {{!-- Header section --}}
+  <UiKit::Box
+    @tag='section'
+    @margin={{hash top=3 bottom=16}}
+    @textAlign='center'
+  >
+    <UiKit::Font @tag='h1' @size='6xl' @weight='bold'>
+      Built for every team
+    </UiKit::Font>
+    <p>
+      Set up your first project in just one minute.
+      <br />
+      Your first 100 builds are free! No credit card required.
+    </p>
+  </UiKit::Box>
+
+  {{!-- Product section --}}
+  <UiKit::Box @tag='section' @margin={{hash bottom=16}}>
+    <UiKit::Grid @alignItems='center' @justify='start'>
+      <UiKit::Font @transform='uppercase' @weight='bold' @margin={{hash right=2}}>
+        Monthly
+      </UiKit::Font>
+
+      <UiKit::Switch @onClick={{fn (mut showAnnual)}} @active={{showAnnual}} />
+
+      <UiKit::Font @color='blue-400' @margin={{hash left=2}}>
+        <UiKit::Font @transform='uppercase' @weight='bold'>Annual</UiKit::Font>
+        (Get 1 month free!)
+      </UiKit::Font>
+    </UiKit::Grid>
+
+    <UiKit::Box @position={{hash type='relative'}} @margin={{hash top=3}}>
+      <UiKit::Box
+        @position={{hash type='absolute' top=0 right=0}}
+        @margin={{hash top=-16 right=-10}}
+        @height={{64}}
+      >
+        <SvgImage @name="grey-blob" @height={{224}} />
+      </UiKit::Box>
+
+      <UiKit::Box
+        @color='white'
+        @layer='10'
+        @overflow='hidden'
+        @position={{hash type='relative'}}
+        @radius='base'
+        @shadow='base'
+      >
+        <UiKit::Grid @base={{2}} @md={{4}} as |grid|>
+          {{#each plansToDisplay as |plan|}}
+            <grid.item>
+              <UiKit::Box @borderColor='grey-150' @borderWidth={{1}} @padding={{hash all=4}}>
+                <UiKit::Font @tag='h3' @size='3xl' @weight='bold'>
+                  {{plan.name}}
+                </UiKit::Font>
+
+                <UiKit::Font
+                  @tag='h4'
+                  @color='grey-400'
```

Comment:
The same about color - let's try to keep it abstract and limited to predefined values.

Reply:
Sounds good, change made :)